var/spack/repos/builtin/packages/aluminum/package.py

```diff
@@ -3,6 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import os
+import sys
```

Comment:
flake is complaining about this unused import

Reply:
Fixed.